api/actions/models.py

```diff
@@ -87,7 +75,9 @@ def has_interest(self):
 
     @property
     def is_assigned(self):
-        return self.action_status == ActionStatus.ASSIGNED
+        return self.action_status == ActionStatus.ASSIGNED \
```

Comment:
I think we should add the ONGOING status to that list too. But maybe a faster check would be `self.assigned_volunteer != None` ?